src/router.tsx

```
@@ -323,6 +334,26 @@ const LayoutEnhanced = () => {
+          dispatch(
+            nodeActionsAsync.getMinimumNetworkProbabilityThunk({
+              apiEndpoint,
+              apiToken: apiToken ? apiToken : '',
+            }),
+          );
+          dispatch(
+            nodeActionsAsync.getSessionsThunk({
+              apiEndpoint,
+              apiToken: apiToken ? apiToken : '',
+              protocol: 'udp'
+            }),
```

ESLint

[error] Missing trailing comma.

(comma-dangle)